package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
+    "test": "karma start karma.conf.js",
```

Comment:
Add two npm script tasks. One should be test:server, another one should be test:client